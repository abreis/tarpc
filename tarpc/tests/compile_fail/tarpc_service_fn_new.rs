#![allow(incomplete_features)]
#![feature(generic_associated_types)]

#[tarpc::service]
trait World {
    async fn new();
}

fn main() {}
