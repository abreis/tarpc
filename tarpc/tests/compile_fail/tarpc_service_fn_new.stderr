error: method name conflicts with generated fn `WorldClient::new`
 --> $DIR/tarpc_service_fn_new.rs:6:14
  |
6 |     async fn new();
  |              ^^^
